### ISSUE TITLE
bore, drill

### ISSUE THREAD

**Issue by arademaker** [globalwordnet/english-wordnet#131]
http://wnpt.sl.res.ibm.com/wn/synset?id=01443021-v

the verb `drill` is hyponym of `cut`?

```
bore, drill
       => cut
           => separate, disunite, divide, part
               => move, displace
```

Should we promote `drill, bore` to hyponym of `displace` (01850315-v)?

**Comment by restinplace** [comment]
Uh, it's a troponym, no?  And like the other tropynyms of cut 
- mortise, tap, incise, plane, etc. -- boring and drilling entail cutting 
with an edged tool, no?   Imho, this is a good example of the 
danger of looking at items pairwise or in isolation, without 
considering the direct and indirect consequences of any changes.

**Comment by jmccrae** [comment]
I think it is pretty odd that `ewn-01555301-v` only has 'cut' as its only member but has a much wider sense than most speakers would assign to cut.

The synset here I don't believe is lexicalized in English:

`ewn-01555301-v` (Interlingual Index: i29494)
(v) cut 
> separate with or as if with an instrument "Cut the rope"

Then, the correct thing to do, would be to eliminate this synset and move all of its hyponyms to this synset's hypernym

**Comment by arademaker** [comment]
I didn't understand the comment from @restinplace above. Is he questioning if {bore, drill} is a troponym of {cut}? Regarding the second question "boring and drilling entail cutting with an edged tool, no?" Is it a definition? I didn't find it.  Anyway, not clear what is his opinion.

Regarding @jmccrae comments, there are lots of claims: 

1) "01555301-v ... has a much wider sense than most speakers would assign to cut" 
2) "ewn-01555301-v is not lexicalized in English"
3) "Then, the correct thing to do... "

About (2), so what should be the sense of `cut` in the example `cut the hope`? Cut is hard, it has 42 senses in PWN. About (1-3), this is @jmccrae opinion or a decision for this issue?

TL; DR, meta issue! 

I believe we do not have precise rules for contributing to this project. The problem that worries me is that it is under the organization from the Global Wordnet Association (GWA), so people will understand as a collaborative work maintained and supported by GWA, and it is not what is happening here. 

Sorry if I am returning to a discussion already raised since our paper in the GWC 2019, but I do not see clear arguments and linguistics tests/criteria being mentioned. It is all about opinions and @jmccrae being the one that ended up making the changes in the files, so, deciding which issue should be closed or not and how.

Maybe that is the only way to go, maybe we should move it out of the GWA organization, or maybe the open-source model will not work for a lexical resource. I do appreciate the initiative of @jmccrae and the others who are reading the issues and making comments, but once we have just opinions, it is hard to make stable decisions, even if I respect the knowledge of everyone here (in particular the English native speakers or the ones with better English skills than me).

**Comment by jmccrae** [comment]
I think there is a problem with 'bore, drill' being a troponym of 'cut'. Looking a bit more it seems that `ewn-01555301-v` is intended to be the main sense of 'cut', but it has quite a broad definition. Most other dictionaries seem to separate several distinct (physical) meanings of 'cut':

https://www.lexico.com/definition/cut
https://en.wiktionary.org/wiki/cut
https://dictionary.cambridge.org/dictionary/english/cut
https://www.merriam-webster.com/dictionary/cut

@arademaker makes a good point that it is hard to make these decisions without more reasonable corpus information to help with this decision. There is certainly room for good, sound tools to help make such decisions, perhaps such as the ELEXIS project could provide soon.

I propose then we leave `ewn-01555301-v` alone and move `ewn-01445640-v`'s hypernym to `ewn-01559703-v` as originally proposed by @arademaker.

Also, there is no reasons that someone else couldn't make any of these changes in the files, and I don't have any formal role in leading this project.  I do try to push things on and develop tools to make it easier and put a lot of effort into this. Personally, I think if every decision has a clear motivation and discussion for it, this is, more accountable than any other process available.
